kolibri/content/api.py

```diff
@@ -217,6 +219,20 @@ def filter_kind(self, queryset, name, value):
             return queryset.exclude(kind=content_kinds.TOPIC).order_by("lft")
         return queryset.filter(kind=value).order_by("lft")
 
+    def filter_by_role(self, queryset, name, value):
```

Comment:
So, this would filter regardless of the value passed in here, right?

Reply:
Right, I wasn't sure if for convention we would just not pass that in if we didn't want to filter the `coach_content` or just always be explicit and pass it in.

Reply:
It would be a little obtuse to pass `coach_content: false` as a GET param, I'll admit.